.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+*Please edit the list to your PR scope, then check off the boxes as you go!*
+
+## Contributor Checklist
+
+- [ ] I have broken down my PR scope into the following TODO tasks
+   -  [ ] task 1
+   -  [ ] task 2
+- [ ] I have passed with at least one python version with `circleci local execute`
```

Comment:
- [ ] I have run the tests locally and they passed. (here's how to use [`circleci local execute`]() link???)
- [ ] I have added tests for any new features or bugs I fixed